code/modules/events/event.dm

```diff
@@ -1,15 +1,16 @@
 /datum/event_meta
 	var/name 		= ""
-	var/enabled 	= 1	// Whether or not the event is available for random selection at all
-	var/weight 		= 0 // The base weight of this event. A zero means it may never fire, but see get_weight()
-	var/min_weight	= 0 // The minimum weight that this event will have. Only used if non-zero.
-	var/max_weight	= 0 // The maximum weight that this event will have. Only use if non-zero.
-	var/severity 	= 0 // The current severity of this event
-	var/one_shot	= 0	//If true, then the event will not be re-added to the list of available events
+	var/enabled 	= TRUE	// Whether or not the event is available for random selection at all
```

Comment:
Since you're refactoring the file. Can you make these work with auto doc?
https://github.com/ParadiseSS13/Paradise/blob/master/.github/AUTODOC_GUIDE.md

Reply:
Entire file should be documented now